obspy/io/nordic/core.py

```diff
@@ -322,40 +414,113 @@ def _readheader(f):
     return new_event
 
 
-def read_spectral_info(sfile):
+def _read_origin(line):
+    """
+    Read one origin (type 1) line.
+
+    :param str line: Origin format (type 1) line
+    :return: `~obspy.core.event.Event`
+    """
+    new_event = Event()
+    try:
+        sfile_seconds = line[16:20].strip()
+        if len(sfile_seconds) == 0:
+            sfile_seconds = 0.0
+        else:
+            sfile_seconds = float(sfile_seconds)
+        new_event.origins.append(Origin())
+        new_event.origins[0].time = UTCDateTime(
+            int(line[1:5]), int(line[6:8]), int(line[8:10]),
+            int(line[11:13]), int(line[13:15]), 0, 0) + sfile_seconds
+    except Exception:
+        raise NordicParsingError("Couldn't read a date from sfile")
+    # new_event.loc_mod_ind=line[20]
+    new_event.event_descriptions.append(EventDescription(text=line[21:23]))
+    for key, _slice in [('latitude', slice(23, 30)),
+                        ('longitude', slice(30, 38)),
+                        ('depth', slice(38, 43))]:
+        try:
+            new_event.origins[0].__dict__[key] = float(line[_slice])
+        except ValueError:
+            new_event.origins[0].__dict__[key] = None
+    if new_event.origins[0].depth:
+        new_event.origins[0].depth *= 1000
+    # new_event.depth_ind = line[44]
+    # new_event.loc_ind = line[45]
```

Comment:
Why are those commented?

Reply:
Commented because I don't know where to put them in the Event format, any ideas? They are single character identifies that the user defines.
depth_ind is usually (AFAIK) one of `(' ', 'F', 'S')` where `'F'` marks fixed depth, `'S'` marks that it is a starting value (for inversion I presume).
loc_ind looks like it is either `(' ', '*')` where `*` means "do not locate".

Reply:
The origins have a `depth_type` attribute:

<img width="731" alt="screen shot 2018-01-25 at 23 08 13" src="https://user-images.githubusercontent.com/800487/35415072-a875faea-0224-11e8-8757-ed1c83f56963.png">

But I'm not sure how that would map to the values you mentioned. There is also an `origin_type` field but that does not seem to be applicable here.

<img width="710" alt="screen shot 2018-01-25 at 23 10 08" src="https://user-images.githubusercontent.com/800487/35415171-e812d524-0224-11e8-9c96-493772e383ad.png">


`loc_ind` make little sense to be to be honest. If there is a location it must have been located somehow.

Reply:
Hmm, I'm not sure the `depth_ind` maps well onto those either.

I think `loc_ind` is used to ensure that the location is not re-calculated, e.g. it was calculated once and the user is happy with it, or it is a known location (like a blast) and should be fixed...

Reply:
I also don't really know how to map these things. I guess there just is no fully suitable mapping. The origin's also have an `epicenter_fixed` boolean field which might be suitable.

Reply:
:+1: that sounds like where it might go. Excellent, thanks!